### DISCUSSION TITLE
antv里面依赖lodash会有安全问题

### DISCUSSION THREAD

**Discussion by wangyeky3419** [antvis/G2#6278] [Q&A]
### 问题描述

antv里面依赖lodash 4.17.4会有安全问题，想升级到lodash4.17.21有什么办法或建议

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by pearmini** [comment]
具体是什么技术栈？
